### PR TITLE
Remove references to the map store

### PR DESCRIPTION
**Pull request by ahus1**
The map store is being removed from Keycloak nightly, the storage option is no longer available.

### CODE REVIEW

**Comment by ahus1** [comment]
@kami619 - I've removed the references to the map store. Please review when you have the time. Thanks!

File: doc/kubernetes/modules/ROOT/pages/storage/embedded-infinispan.adoc [hunk 1]
```diff
@@ -1,26 +0,0 @@
-= Configuring Embedded Infinispan
```
Comment (kami619) [inline doc/kubernetes/modules/ROOT/pages/storage/embedded-infinispan.adoc:1]: @ahus1 We would still have the embedded-infinispan as part of the Keycloak store in its current shape, do we want to move this information to a different place instead ?

**Comment by ahus1** [comment]
@kami619 - thanks, re-adding the file. I see that it is not referenced anywhere (?), still that will need to be fixed in another commit.

File: kcb-ci-runner.sh [hunk 2]
```diff
@@ -9,18 +9,8 @@ export RESET_KEYCLOAK=${RESET_KEYCLOAK:-true}
 export KEYCLOAK_STORAGE=${KEYCLOAK_STORAGE:-"JPA-Legacy-PostgreSQL"}
 if $RESET_KEYCLOAK; then
   cd $MINIKUBE_HOME
-  if [ "$KEYCLOAK_STORAGE" = "JPA-Map-PostgreSQL" ]; then
-    task reset-keycloak KC_STORAGE="jpa" KC_DATABASE="postgres"
-  elif [ "$KEYCLOAK_STORAGE" = "JPA-Legacy-PostgreSQL" ]; then
+  if [ "$KEYCLOAK_STORAGE" = "JPA-Legacy-PostgreSQL" ]; then
```
Comment (kami619) [inline kcb-ci-runner.sh:12]:
```suggestion
  if [ "$KEYCLOAK_STORAGE" = "JPA-PostgreSQL" ]; then
```

I will adjust the Jenkins Job thats consuming this shell script.

File: kcb-ci-runner.sh [hunk 3]
```diff
@@ -9,18 +9,8 @@ export RESET_KEYCLOAK=${RESET_KEYCLOAK:-true}
 export KEYCLOAK_STORAGE=${KEYCLOAK_STORAGE:-"JPA-Legacy-PostgreSQL"}
```
Comment (kami619) [inline kcb-ci-runner.sh:9]:
```suggestion
export KEYCLOAK_STORAGE=${KEYCLOAK_STORAGE:-"JPA-PostgreSQL"}
```

File: provision/common/Taskfile.yaml [hunk 4]
```diff
@@ -8,7 +8,6 @@ vars:
   KC_DB_POOL_INITIAL_SIZE: '{{default "5" .KC_DB_POOL_INITIAL_SIZE}}'
   KC_DB_POOL_MAX_SIZE: '{{default "10" .KC_DB_POOL_MAX_SIZE}}'
   KC_DB_POOL_MIN_SIZE: '{{default "5" .KC_DB_POOL_MIN_SIZE}}'
-  KC_STORAGE: '{{default "" .KC_STORAGE}}'
```
Comment (kami619) [inline provision/common/Taskfile.yaml:11]: so we are effectively removing the `KC_STORAGE` as an `.env` option ? I think that makes sense.

Comment (ahus1) [reply provision/common/Taskfile.yaml:11]: Yes, it is being removed.

File: provision/minikube/keycloak/templates/keycloak.yaml [hunk 5]
```diff
@@ -71,27 +65,6 @@ spec:
...
-      value: jpa
-    - name: storage-area-auth-session
-      value: hotrod
-    - name: storage-area-user-session
-      value: hotrod
-    - name: storage-area-single-use-object
-      value: hotrod
-    - name: storage-area-login-failure
-      value: hotrod
-{{ else if ne .Values.storage "" }}
```
Comment (kami619) [inline provision/minikube/keycloak/templates/keycloak.yaml:87]: Would this configuration be not valid anymore ?

File: doc/kubernetes/modules/ROOT/pages/storage-configurations.adoc [hunk 6]
```diff
@@ -3,10 +3,7 @@
 
 {description}
 
-Keycloak provides two storage implementations:
-
-* Legacy Storage
-* New Storage (also called Map Storage)
+This setup supports different databases.
```
Comment (kami619) [inline doc/kubernetes/modules/ROOT/pages/storage-configurations.adoc:6]: currently it doesn't, as it only supports `postgres` right @ahus1 ?
```suggestion
This setup supports only PostgreSQL database.
```

File: provision/minikube/keycloak/templates/keycloak.yaml (hunk 5 above)
Comment (ahus1) [reply provision/minikube/keycloak/templates/keycloak.yaml:87]: All storage configurations are removed, so it is no longer valid.

File: doc/kubernetes/modules/ROOT/pages/storage-configurations.adoc (hunk 6 above)
Comment (ahus1) [reply doc/kubernetes/modules/ROOT/pages/storage-configurations.adoc:6]: I counted Aurora as a second database :-)

**Review by kami619 — APPROVED** [review]
thanks @ahus1 for this extensive change with the map store removal, this should def save us some big chunks of time on PR CI jobs.

**Pull request merged**